src/components/TextStyle/tests/TextStyle.test.tsx

```diff
@@ -33,6 +33,13 @@ describe('<TextStyle />', () => {
     expect(textStyle).toContainReactComponent('span');
   });
 
+  it('renders a span when the variant warning is provided', () => {
+    const textStyle = mountWithApp(
+      <TextStyle variation="warning">Hello Polaris</TextStyle>,
+    );
+    expect(textStyle).toContainReactComponent('span');
```

Comment:
What kind of tests are these 😂

Reply:
I have no idea... literally had the same thought. Should we add a test for the right class? Or just remove these tests?

Reply:
Leave it in for now. We can remove all of these, or leave one, and rely on visual regression testing but we'll decide that as a team. I'll open an issue to follow up with